bitnami/sonarqube/values.yaml

```diff
@@ -900,28 +900,24 @@ postgresql:
     ## @param postgresql.auth.database Database to create when deploying the PostgreSQL chart
     ##
     database: bitnami_sonarqube
-
-  ## @param primary.name Name of the primary database (eg primary, master, leader, ...)
-  ##
-  primary:
   ## PostgreSQL service
   ##
   service:
-  ## @param postgresql.primary.service.ports.postgresql PostgreSQL service port
-  ##
+    ## @param postgresql.service.ports.postgresql PostgreSQL service port
+    ##
```

Comment:
The tabulations are needed
```
  ## @param primary.name Name of the primary database (eg primary, master, leader, ...)
  ##
  primary:
    ## PostgreSQL service
    ##
    service:
    ## @param postgresql.primary.service.ports.postgresql PostgreSQL service port
    ##
.....
```
Could you add it in the rest of the values that have primary?